### DISCUSSION TITLE
Is there a way to read data from internal database through REST api?

### DISCUSSION THREAD

**Discussion by Korayem** [Budibase/budibase#3638] [Ideas]
I want to build a CMS management for my content team which will be consumed by frontend/mobile apps

Would be great if budibase can auto create CRUD APIs for Internal Database entities. This will make budibase act like XANO.

**Comment by shogunpurple** [comment]
Hi @Korayem - you can find more about this in this discussion: https://github.com/Budibase/budibase/discussions/1915

We don't have a fully documented REST API yet, but will do in the coming months. Hopefully this solves your problem in the meantime.

Thanks!
